src/validations/rules/sap.sch

```diff
@@ -478,6 +477,20 @@
                 test="oscal:part[@name eq 'disclosure']">The SAP Rules of Engagement (ROE) Disclosures have one or more detail disclosure
                 statements.</sch:assert>
 
+            <sch:let
+                name="unsorted_disclosures"
+                value="oscal:part[@name eq 'disclosure']/oscal:prop[@name eq 'sort-id']/@value" />
+            <sch:let
+                name="sorted_disclosures"
```

Comment:
We have a mix between camelCase, dash-case, and snake_case for variables. Might be good to standardize. (not suggesting do that now)

Reply:
I agree Dan.  I'll bet most of that is due to 3+ developers on the project.  I know I default to camelCase when I am not keeping dash in mind.

I'll add an issue to READY to check all variables across the schematron near the end of the project.